pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml

```diff
@@ -114,6 +114,12 @@ spec:
             - "--forcecephkernelclient={{ .ForceCephFSKernelClient }}"
             - "--metricspath=/metrics"
             - "--enablegrpcmetrics={{ .EnableCSIGRPCMetrics }}"
+            {{ if .CSIEnableMetadata }}
```

Comment:
Do we need this if check? if `CSIEnableMetadata` is false we will not go inside loop and we cannot set `setmetadata` to false

Reply:
setmetadata is false by default at CSI level, do you see anything wrong in not setting --setmetadata=false?

Reply:
If for now, it will work what if it is enabled by default  at the CSI level and still the user wants to disable it? one case might be the user still using some external ceph cluster

Reply:
Sure, this make sense to me.